packages/app-config-writer/README.md

```diff
@@ -1,6 +1,12 @@
 # @sap-ux/app-config-writer
 
-The `@sap-ux/app-config-writer` adds smartlinks configuration for a SAP UX project. 
+The `@sap-ux/app-config-writer` adds or updates configurations to a SAP UX project.
+
+## Features
+
+### Smart Links configuration
+Smart Link controls can be defined in an application to provide navigation links to other Fiori Applications in a standardized way.
+To deliver an application with pre-configured targets for the included smart links, it is necessary to add a smartlinks configuration to the project.
```

Comment:
This makes it sound that it is required for the productive use of the app, however, based on the changes, I assumed, it is only required for local testing. Please clarify.

Reply:
Thanks, clarified and updated the readme with the changes